auth/realm/base/src/main/java/org/wildfly/security/auth/realm/LegacyPropertiesSecurityRealm.java

```diff
@@ -457,6 +461,7 @@ public Builder setGroupsAttribute(final String groupsAttribute) {
          * @return this {@link Builder}
          */
         public Builder setDefaultRealm(String defaultRealm) {
+            assertNotNull(defaultRealm);
```

Comment:
Looks like CI is hitting a test failure related to this method. `null` is a valid value here, so we can remove the null check for this one.

Reply:
Fixed it.